.travis.yml

```diff
@@ -71,12 +71,11 @@ script:
   - mkdir build && pushd build &&
     cmake -DCMAKE_INSTALL_PREFIX="$HOME/cabana"
           -DCMAKE_CXX_COMPILER_LAUNCHER=ccache
-          -DKokkos_INSTALL_PATH="$HOME/kokkos"
+          -DCMAKE_PREFIX_PATH="$HOME/kokkos"
           -DCabana_REQUIRE_SERIAL=OFF ${CMAKE_OPTS[@]}
-          -DCabana_ENABLE_MPI=ON
+          -DCabana_REQUIRE_MPI=ON
           $([[ ${CC} = clang* ]] || echo -DCabana_ENABLE_FORTRAN_EXAMPLES=ON)
           -DCabana_ENABLE_TESTING=ON -DCabana_ENABLE_EXAMPLES=ON
-          -DCabana_ENABLE_DOXYGEN=ON
```

Comment:
Why?

Reply:
That's not an option anymore - we were always checking for doxygen in every build.

Reply:
Nice!